### PR TITLE
docs: proof-read fast-builds nearcore book page

### PR DESCRIPTION
**Pull request by nagisa**
This all started with me noticing a typo ("to slow”), but then I realized there are other places that could be reworded to flow smoother as well.

### CODE REVIEW

File: docs/practices/fast_builds.md [hunk 1]
```diff
@@ -1,34 +1,36 @@
...
 nearcore is implemented in Rust and is a fairly sizable project, so it takes a
-while to build. This chapter collects various tips to make the process faster.
+while to build. This chapter collects various tips to make the development
+process faster.
 
 Optimizing build times is a bit of a black art, so please do benchmarks on your
-machine to verify that the improvement work for you. Changing some configuration
+machine to verify that the improvements work for you. Changing some configuration
 and making some type, which prevents it from improving build times is an
 extremely common failure mode!
```
Comment (nagisa) [inline docs/practices/fast_builds.md:10]: I’m not quite sure what the last sentence is trying to say here. What does "and making some type” mean and the way it affects build times doesn't really connect.

Comment (matklad) [reply docs/practices/fast_builds.md:10]: X_X that was supposed to be "some typo"

File: docs/practices/fast_builds.md [hunk 2]
```diff
@@ -1,34 +1,36 @@
...
 
 nearcore is implemented in Rust and is a fairly sizable project, so it takes a
-while to build. This chapter collects various tips to make the process faster.
+while to build. This chapter collects various tips to make the development
+process faster.
 
 Optimizing build times is a bit of a black art, so please do benchmarks on your
-machine to verify that the improvement work for you. Changing some configuration
+machine to verify that the improvements work for you. Changing some configuration
 and making some type, which prevents it from improving build times is an
```
Comment (matklad) [inline docs/practices/fast_builds.md:9]:
```suggestion
and making a typo, which prevents it from improving build times is an
```

**Review by matklad — APPROVED** [review]
:heart_eyes:

**Pull request merged**